src/transaction/transaction_slave_group_complete_manager.hpp

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (C) 2008 Search Solution Corporation. All rights reserved by Search Solution.
+ *
+ *   This program is free software; you can redistribute it and/or modify
+ *   it under the terms of the GNU General Public License as published by
+ *   the Free Software Foundation; either version 2 of the License, or
+ *   (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with this program; if not, write to the Free Software
+ *  Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+ *
+ */
+
+//
+// Manager of completed group on a HA slave node
+//
+
+#ifndef _TRANSACTION_SLAVE_GROUP_COMPLETE_MANAGER_HPP_
+#define _TRANSACTION_SLAVE_GROUP_COMPLETE_MANAGER_HPP_
+
+#include "transaction_group_complete_manager.hpp"
+
+#include "cubstream.hpp"
+#include "log_consumer.hpp"
+#include "thread_daemon.hpp"
+#include "thread_entry_task.hpp"
```

Comment:
`#include "thread_entry_task.hpp"` no longer needed here. probably it must be included in cpp file.